eth2deposit/intl/en/cli/generate_keys.json

```diff
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "Create a password that secures your validator keystore(s)",
+        "msg_password_confirm": "Repeat for confirmation",
+        "err_password_mismatch": "Error: the two entered values do not match. Please type again."
+    },
+    "generate_keys_arguments_decorator": {
+        "num_validators": {
+            "argument": "--num_validators",
+            "help": "The number of validators keys you want to generate (you can always generate more later)",
+            "prompt": "Please choose how many validators you wish to run"
+        },
+        "folder": {
+            "argument": "--folder",
+            "help": "The number of validators keys you want to generate (you can always generate more later)"
+        },
+        "chain": {
+            "argument": "--chain",
+            "help": "The version of eth2 you are targeting. Use \"mainnet\" if you are depositing ETH",
```

Comment:
Just realized that "the version of eth2" might be a bit off. Maybe just "The network/chain name you are targeting"?

Probably to be addressed in separate PR.